tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/Matter/MatterRepository.cs

```diff
@@ -1289,10 +1305,21 @@ public bool SetItemPermission(ClientContext clientContext, IList<IList<string>>
                     #region Remove all the external users from the request object so that only internal users are added to the matter
                     //Once the external users accepted the invitation, those external users will be added to the matter by azure web app job
                     List<int> itemsToRemove = new List<int>();
+                    ///Below changes are required for the matters where no roles are available in the stamped properties. By default we are adding Responsible Attorney
+                    if (generalSettings.IsBackwardCompatible == true)
```

Comment:
If there are no roles present for old matters, we want to add "Responsible Attorney" as the default roles for all the users associated with that matter